docker/synapse3_10.Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG SYTEST_IMAGE_TAG=bookworm
```

Comment:
we could do with a comment here. What does this file do, how is it different from `synapse.Dockerfile`, etc?